pkgs/by-name/op/opengfw/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+let
+  pname = "opengfw";
+  version = "0.4.0";
+in
+buildGoModule rec {
```

```suggestion
buildGoModule {
```

We are already using a `let` binding here so `rec` is not needed